compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__shadowed_fn_argument.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: compiler-core/src/type_/tests/errors.rs
+expression: "\npub fn go(x) {\n  fn(_y) {\n    y + x\n  }\n}\n"
+---
+----- SOURCE CODE
+
+pub fn go(x) {
+  fn(_y) {
+    y + x
+  }
+}
+
+
+----- ERROR
+error: Unknown variable
+  ┌─ /src/one/two.gleam:4:5
+  │
+3 │   fn(_y) {
+  │      -- Did you mean to use this ignored variable?
+4 │     y + x
+  │     ^
```

Comment:
This seems a bit confusing as the actual error location is less prominent and comes after the suggestion for how to fix it.

Perhaps something like this?

```
error: Unknown variable
  ┌─ /src/one/two.gleam:4:5
  │
3 │   fn(_y) {
  │      -- This value is discarded
4 │     y + x
  │     ^ So this is not in scope

Hint: Change `_y` to `y` or reference another variable.
```

Reply:
this looks amazing